### ISSUE TITLE
Can you please also create a docker image for ZimaOS?

### ISSUE THREAD

**Issue by serycoola** [dockur/umbrel#14]

**Comment by serycoola** [comment]
Sorry, I think my phrasing was a little bit off. I didn't mean to create a special image for use with ZimaOS. This one works just fine.

I wanted to ask you (but I had no idea how to contact you, besided the issues page) if you could please create a similar docker image, but with ZimaOS instead of Umbrel. ZimaOS, just like Umbrel, moved from an install script to their own distro, and because of this it would be very helpful if there would also be a docker image of it. 

I personally need that because their kernel doesn't have drivers for my network card, and also they have no package manager inside their distro. 

I tried following a guide written by ChatGPT to do that by myself, but I am unsuccessful. I could give you the tarball of the installed ZimaOS system, if it would help you make this easier. Or, at the very least, would you mind giving me a hand to be able to set this up by myself? I am at the point where docker run -it zimaos-image works and puts me into bash, but I don't know how to set it up to also load anything inside the browser...

**Comment by serycoola** [comment]
Yes, ZimaOS is CasaOS with extra features, but CasaOS will never receive any newer features, while ZimaOS already has some interesting ones. Namely, the ability to share a file via link, so that you can share something with someone without the need to expose the server to the internet. And they also added VMs and redesigned the built in file manager.

Would it be difficult to create a docker image of this OS?

**Comment by kroese** [comment]
I now created a Docker container of ZimaOS: [dockur/zima: ZimaOS inside a Docker container.](https://github.com/dockur/zima)
